circle.yml

```diff
@@ -50,17 +50,12 @@ jobs:
           name: Deploy
           command: npm publish
           # Output used for publish is from node 4 build
```

Comment:
I guess the "node 4" comment here needs updating ;-)

Reply:
well spotted :)